c7n/filters/securityhub.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2017 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from c7n.utils import local_session, chunks, type_schema
+from .core import Filter
+from c7n.manager import resources
+
+
+class SecurityHubFindingFilter(Filter):
+    """Check if there are Security Hub Findings related to the resources
+    """
+    schema = type_schema('findings')
```

Comment:
while not universal we typically use singular form, ie `finding`

Reply:
yeah preference would be singular, just checked all the ec2 filters, and their all singular form.

```
custodian schema ec2.filters                                                                                                                                                                                                                                             
aws.ec2:
  filters: [and, config-compliance, default-vpc, ebs, ephemeral, event, health-event,
    image, image-age, instance-age, instance-attribute, instance-uptime, marked-for-op,
    metrics, network-location, not, offhour, onhour, or, security-group, singleton,
    state-age, subnet, tag-count, termination-protected, user-data, value, vpc]
```